---
layout: frontpage
title: Lobo et al. (2021) Fig 1
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="mbmixups_fig3.html">prev</a></li>
          <li><a href="iplotCorr.html">next</a></li>
      </ul>
  </div>
</div>

Lobo AK, Traeger LL, Keller MP, Attie AD, Rey FE, **Broman KW** (2021)
Identification of sample mix-ups and mixtures in microbiome data in
Diversity Outbred mice. [G3
(Bethesda)](https://academic.oup.com/g3journal), to appear
[![pdf](../icons16/pdf-icon.png)](https://academic.oup.com/g3journal/advance-article-pdf/doi/10.1093/g3journal/jkab308/40259169/jkab308.pdf)
[![GitHub](../icons16/github-icon.png)](https://github.com/kbroman/Paper_MBmixups)
[![R/mbmixture software](../icons16/R-icon.png)](https://github.com/kbroman/mbmixture)
[![doi](../icons16/doi-icon.png)](https://doi.org/10.1093/g3journal/jkab308)

![Lobo et al. (2021) Fig 1](../bigpublpics/mbmixups_fig1_lg.png)

**Figure 1**. &mdash; Plot of minimum distance vs. distance to self. Each point is a
microbiome sample and distances to genomic DNA samples are measured
by taking, among reads that overlapped a SNP where the genotyped
sample was homozygous, the proportion with an allele that was
discordant with the inferred SNP genotype. The microbiome samples
are categorized according to our conclusions about their
status.
